edge/test/integration/scripts/execute.sh

```diff
@@ -19,23 +19,20 @@ cd $workdir
 
 curpath=$PWD
 echo $PWD
-
-if [ ! -d "/var/lib/edged" ]; then
```

Comment:
Please move the e2e change out of this PR, why is not urgent

Reply:
Because the KubeEdge component now used the new configuration file, e2e test is related to the configuration file. If it is not modified, e2e test will fail